test/e2e-common.sh

```diff
@@ -15,8 +15,9 @@
 # limitations under the License.
 
 # This script provides helper methods to perform cluster actions.
-source $(dirname $0)/../vendor/knative.dev/hack/e2e-tests.sh
-source $(dirname $0)/e2e-networking-library.sh
+# shellcheck disable=SC1090
+source "$(dirname "${BASH_SOURCE[0]}")/../vendor/knative.dev/hack/e2e-tests.sh"
+source "$(dirname "${BASH_SOURCE[0]}")/e2e-networking-library.sh"
```

Comment:
Let's please do changes like these separately. they unnecessarily bloat the diff and have nothing to do with the intended change as far as I can tell?

I believe this applies to most of the bash changes?

Reply:
If this is required for the shell shenanigans, let's still break the bash changes out to break this up a little.

Reply:
This is required so that we can call e2e-common.sh from Golang. Without that the file cannot be found. I don't think this can be sent separately.

Reply:
This can be sent separately, but that has only little sense. As, apart from this use case previous code works, so without changes in here it's hard to justify changing that. Maybe say it's required for this PR, or it may error if sourced from different place than `test/` directory.